tests/end_to_end/L1L2Messaging/test_messaging.py

```diff
@@ -73,14 +89,22 @@ def message_app_l1(sn_messaging_local, l1_kakarot_messaging, kakarot):
 
 
 @pytest.fixture(scope="function")
-def wait_for_message(sn_messaging_local):
+def wait_for_message_l1(message_app_l1, sn_messaging_local):
```

Comment:
This will be duplicated code. In the end the fixture is here to wait for L1 messages. Now we wait for 2 types of message, no ?
But if you think this is better to separate those 2, I am open to make the change

Reply:
in fact this is better to use 2 fixtures. Clearer.
Will push the fixes